src/main.ts

```diff
@@ -251,6 +282,8 @@ export abstract class BaseFargateService extends cdk.Construct {
         circuitBreaker: props.circuitBreaker != false ? {
           rollback: true,
         } : undefined,
+        maxHealthyPercent: t.maxHealthyPercent ? t.maxHealthyPercent : undefined,
+        minHealthyPercent: t.minHealthyPercent ? t.minHealthyPercent : undefined,
```

Comment:
```suggestion
        maxHealthyPercent: t.maxHealthyPercent,
        minHealthyPercent: t.minHealthyPercent,
```

I think we can just write it this way.